index.html

```diff
@@ -171,7 +173,7 @@ <h2 class="myself">About me</h2>
        </div>
         <div class="Frameworks1">
           <ul id="list">
-            <li class="listFRAM"><img src="img/Rectangle 55@3x.png" alt="" class="image4" >
+            <li class="listF"><img src="img/Rectangle 55@3x.png" alt="" class="image4" >
               <p class="langQ1">Frameworks</p></li>
             <li class="lang1">Bootstrap</li>
             <li class="lang2">Ruby on Rails</li>
```

Comment:
- [ ] The same thing that affected your about me section also affects this section. Kindly look into the unwanted space to the right of the section and remove it so as to make your project match the guiding template. You also need to work on the alignment of the list of languages and frameworks so as to look exactly like how they appear on the figma design.

### Your Design
[your-lang-frame.webm](https://user-images.githubusercontent.com/26597432/206870141-42271d2e-30ad-42e0-9625-61d272dcc94a.webm)

### Figma Design
![figma-lang-frame](https://user-images.githubusercontent.com/26597432/206870148-7d59b4e8-2da5-4204-acd1-e7074c07cfaa.png)